test/suites/migrations.spec.ts

```diff
@@ -126,4 +126,20 @@ describe('Environments migrations', () => {
       );
     });
   });
+
+  describe('No. 15', () => {
+    const filesPath = 'migrations/15';
+    const tests = new Tests(filesPath);
+
+    it('Should add "proxyRemovePrefix" properties to environments', async () => {
+      await tests.helpers.waitForAutosave();
+
+      await tests.helpers.verifyObjectPropertyInFile(
+        './tmp/storage/environments.json',
+        ['0.proxyRemovePrefix'],
+        false,
+        true
```

Comment:
I think you can remove the last param `true` as we want to test if it's `false`.

Reply:
I interpreted those parameters differently, but digging into that function I agree it should be pulled out. I updated the branch.